src/libical-glib/i-cal-object.c.in

```diff
@@ -324,7 +324,7 @@ static void i_cal_object_init(ICalObject *iobject)
  * @native: a native libical structure
  * @native_destroy_func: a function to be called on @native when it should be freed
  * @is_global_memory: whether @native is a global shared memory structure
- * @owner: (allow-none): an owner of @native
+ * @owner: (nullable): an owner of @native
```

Comment:
Might it be `optional` instead? I'm still confused about the two, especially because they both can be applied to the argument at the same time.

Reply:
optional can only be used with `(out)` and `(inout)`
for instance:
`const gchar **var`
(optional) means that `var == NULL` is allowed while (nullable) means that `*var` can be set to NULL by the function